### DISCUSSION TITLE
RSVP feature

### DISCUSSION THREAD

**Discussion by PagnozziM** [pretix/pretix#4164] [Ideas] [answered]
Hello!
I though to briefly write down here the use case we're dealing with here in the company I'm working for.

Most of the events we run with the help of Pretix are internal meetings for which we need to know both who is attending and who wishes not to attend.
Because of this need (and because, at the moment, I saw there is no other way to manage this), I managed to 

- invite potential attendants sending them a newsletter
- in the newsletter, I prepared 2 buttons - one for "Yes, I'll participate" (which points directly to Pretix), one for "No, I'll not participate"
- the "No" button points to a survey system (LimeSurvey), placed "in front of" ("before") Pretix, and set in order to be totally transparent and to capture the "no" answer, then to redirecting the user to an external landing page ("Thanks for answer" or similar).

In this way:

- who decided to participate, arrives to Pretix and performs registration
- who decided not to participate, goes away but I'm able to make differentiation between who said "no" and who simply didn't replied.

If Pretix could have an "RSVP function" everything would be easier and, most importantly, the totality of information would be available in one place.
I have seen that systems similar to Pretix have this function.

What are your thoughts on this?

Thank you for your attention.
Kind regards.

Michele Pagnozzi

**Comment by raphaelm** [comment]
Yes, we have this as a long-term goal. We're still looking for the perfect approach though (and have some other priorities first), but I'm pretty sure it will happen in the next 1-3 years either as a plugin or in core.

**Reply by PagnozziM** [reply]
Hi @raphaelm ,
Just for my information, have there been any developments on this topic?

Thanks,
Kind regards,
MP

**Reply by raphaelm** [reply, marked as answer]
No, it's currently not one of our priority topics
